extensions/interactions/Continue/Continue.py

```diff
@@ -34,3 +34,5 @@ class Continue(base.BaseInteraction):
         },
         'default_value': 'Continue',
     }]
+
+    _answer_visualization_specs = []
```

Comment:
Instead of defining this in each interaction, you could define it in the base interaction class and only specify it if it's not an empty list. I think that will read better and make it easier on future interaction authors.